packages/core/src/App/Containers/RealAccountSignup/personal-details-form.js

```diff
@@ -94,19 +94,23 @@ export const personalDetailsConfig = ({ can_upgrade_to }) => {
             ],
             salutation_list: [
                 {
-                    text: localize('Mr'),
+                    id: 'Mr',
```

Comment:
What is the reason behind adding an additional key here?

Reply:
Each radio has a uniqe id, if we wanna select a radio in DOM then there is a id for that, but `name` is the same in all radio elements.

Reply:
As @salar-fs mentioned, value can be re-used here as an ID here.